lib/src/showcase.dart

```diff
@@ -327,50 +325,3 @@ class _ShowcaseState extends State<Showcase> {
         : SizedBox.shrink();
   }
 }
-
-class _TargetWidget extends StatelessWidget {
```

Comment:
what is the reason for removing this widget ?

Reply:
The use of _TargetWidget is only to create a container having size of the child and perform onTargetTap action on that container.

This Widget is only used in one place which is no longer needed. ( at 298th line)

Reply:
please consider to include this widget as it provider gesture to overlay padding.